andy/src/test/resources/grader/fixtures/Config/NumberUtilsAddConfigurationInvalid.java

```diff
@@ -0,0 +1,100 @@
+package delft;
+
+import nl.tudelft.cse1110.andy.config.RunConfiguration;
+import nl.tudelft.cse1110.andy.config.MetaTest;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Configuration extends RunConfiguration {
```

Comment:
I think we still need an integration test, with a wrong configuration, and then, we assert that the produced `Result` contains the error message. (See other integration tests in the integration folder)

Reply:
This should be the wrong configuration for one of the tests. Maybe I am understanding it wrong?

Here is the code for the test:
```java
    @Test
    void checkExceptionHandling() {
        Result result = run("NumberUtilsAddLibrary", "NumberUtilsAddAllTestsPass", "NumberUtilsAddConfigurationInvalid");
        assertThat(result.getGenericFailure().getExceptionMessage().toString().contains("The run configuration has invalid weights. Please check the file.")).isTrue();
    }
```
    
   Found in the test suite

Reply:
Ah ok, I missed the `checkExceptionHandling`. So, I'd split the unit tests from this integration test. Don't put them on the same file, as this isn't how we have been doing :)